src/conq/perception_lib/sam_inference.py

```diff
@@ -17,7 +12,10 @@
 import numpy as np
 import replicate
 
-os.environ['REPLICATE_API_TOKEN'] = '***ADD API TOKEN HERE***'
+from conq.api_keys import REPLICATE_API_KEY
+
+os.environ['REPLICATE_API_TOKEN'] = REPLICATE_API_KEY.get()
+
 
 def run_sam_inference(image_url):
     start_time = time.time()
```

Comment:
@Saketspradhan two lines below this one (so, line 23), it looks like there's an API key hard-coded. Is that one?

Reply:
Which file/line are you referring to? Is it something to do with the Roboflow API?

Reply:
```
def run_sam_inference(image_url):
    start_time = time.time()
    image_url = replicate.run(
        "pablodawson/segment-anything-automatic:14fbb04535964b3d0c7fad03bb4ed272130f15b956cbedb7b2f20b5b8a2dbaa0",
        input={
            "image": image_url
        }
    )
```

Are you referring to the `hash-key` in this block?

Reply:
That's not a problem, it is public access within replicate itself

Reply:
Yep, that's what I was referring to. Sounds like it's fine to have it in here so I'll resolve this conversation.